<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderSourceEditDialog</class>
 <widget class="QDialog" name="ShaderSourceEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit your shader source</string>
  </property>
  <widget class="QTextEdit" name="shaderEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="rejectButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>145</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="vertexShaderRadioButton">
      <property name="text">
       <string>Vertex shader</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fragmentShaderRadioButton">
      <property name="text">
       <string>Fragment shader</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="geometryShaderRadioButton">
      <property name="text">
       <string>Geometry shader</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
